src/App.test.jsx

```diff
@@ -1,6 +1,11 @@
 import React from "react";
 import { shallow } from "enzyme";
 import App from "./App";
+import FAQ from "./pages/FAQ";
+import Information from "./pages/Information";
+import Profile from "./pages/Profile";
+import Registration from "./pages/Registration";
+import AccordionList from "./components/AccordionList";
```

Comment:
:wrench: Rebase this branch and remove these lines.

Reply:
Addressed:  a379eb7b3721a74ff9e1e84d867dfc18c86eb75b